vignettes/standardize_data.Rmd

```diff
@@ -15,7 +15,6 @@ vignette: >
 library(knitr)
 options(knitr.kable.NA = "")
 knitr::opts_chunk$set(
-  comment = ">",
```

Comment:
Shouldn't we add `comment = "#>",` instead?

That would also remove ANSI escape sequences from showing up, IINM.

Reply:
done